### PR TITLE
Np jprb pd 2353 multiple star rsolo align

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

_Give your PR a **concise** yet **descriptive** title._
_Please explain the changes you made here._
_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
_Mention any issues fixed, addressed, or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._
_You can delete these instructions once you have written your PR description._

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by kevinpalis — APPROVED** [review]
LGTM

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 6.3.3
+2024-01-05 (Date of Last Commit)
+* Modified the STARsoloFastq task in the StarAlign.wdl so STARsolo can run different types of alignments in a single STARsolo command depending on the counting_mode.
```
Comment (ekiernan) [inline pipelines/skylab/optimus/Optimus.changelog.md:3]:
```suggestion
* Modified the STARsoloFastq task in the StarAlign.wdl so STARsolo can run different types of alignments in a single STARsolo command depending on the counting_mode

File: pipelines/skylab/paired_tag/PairedTag.changelog.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # 0.0.3
-2024-01-02 (Date of Last Commit)
+2024-01-05 (Date of Last Commit)
 
 * Added a new option for the preindex boolean to add cell barcodes and preindex sample barcode to the BB tag of the BAM
-* Added new functionality for the ATAC workflow to use BB tag of BAM for SnapATAC2 
+* Added new functionality for the ATAC workflow to use BB tag of BAM for SnapATAC2
+* Modified the STARsoloFastq task in the StarAlign.wdl so STARsolo can run different types of alignments in a single STARsolo command depending on the counting_mode.
```
Comment (ekiernan) [inline pipelines/skylab/paired_tag/PairedTag.changelog.md:6]:
```suggestion
* Modified the STARsoloFastq task in the StarAlign.wdl so STARsolo can run different types of alignments in a single STARsolo command depending on the counting_mode

File: pipelines/skylab/slideseq/SlideSeq.changelog.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# 2.1.4
+2024-01-05 (Date of Last Commit)
+* Modified the STARsoloFastq task in the StarAlign.wdl so STARsolo can run different types of alignments in a single STARsolo command depending on the counting_mode; this does affect the SlideSeq workflow.
```
Comment (ekiernan) [inline pipelines/skylab/slideseq/SlideSeq.changelog.md:3]:
```suggestion
* Modified the STARsoloFastq task in the StarAlign.wdl so STARsolo can run different types of alignments in a single STARsolo command depending on the counting_mode; this does affect the SlideSeq workflow

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+# 1.2.26
+2024-01-05 (Date of Last Commit)
+* Modified the STARsoloFastq task in the StarAlign.wdl so STARsolo can run different types of alignments in a single STARsolo command depending on the counting_mode; this does affect the MultiSampleSmartSeq2SingleNucleus workflow.
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md:3]:
```suggestion
* Modified the STARsoloFastq task in the StarAlign.wdl so STARsolo can run different types of alignments in a single STARsolo command depending on the counting_mode; this does affect the MultiSampleSmartSeq2SingleNucleus workflow

**Review by ekiernan — APPROVED** [review]
Looks good as long as Smart-tests are passing.

**Pull request merged**